docker-compose.yml

```diff
@@ -123,6 +127,18 @@ services:
     volumes:
       - db_data:/var/lib/postgresql/data:delegated
 
+  testdb:
+    image: postgres:13-alpine
+    container_name: forem_postgresql_test
+    environment:
+      POSTGRES_USER: forem
+      POSTGRES_PASSWORD: forem
+      POSTGRES_DB: Forem_test
+    expose:
```

Comment:
I prefer expose to port, in this case so both the test db and developer db can use the same (default) port, without conflicting. 

This means a user can't connect from the host to the db (you have to do that from the container).